### PR TITLE
Update mysql.md

### PR DESCRIPTION
**Pull request by eljo**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

File: _resources/mysql.md [hunk 1]
```diff
@@ -40,6 +40,13 @@ You can add this URL to any application with `convox env set`:
 
     $ convox env set 'DATABASE_URL=mysql://mysql::)t[THpZ[wmCn88n,N(:@my1.cbm068zjzjcr.us-east-1.rds.amazonaws.com:3306/app' --app example-app
 
+## Resource Update
+
+You can change e.g. multi-zone availablitly with `convox resources update`:
```
Comment (mwarkentin) [inline _resources/mysql.md]: Typo on availability.

Comment (mwarkentin) [inline _resources/mysql.md]: "You can change options like multi-zone..."?

**Comment by soulshake** [comment]
[Review site PR (Update mysql.md #628)](https://trello.com/c/1SIu90lq/2085-review-site-pr-update-mysql-md-628)

**Comment by soulshake** [comment]
I went ahead and made the changes. Thanks for the contribution!

**Pull request merged**